build.gradle

```
@@ -180,6 +181,11 @@ tasks.register("configureArtifactInfo") {
         def dlVersions = new JsonSlurper().parseText(apiResponse)
         String qualifiedVersion = dlVersions['versions'].grep(isReleaseBuild ? ~/^${version}$/ : ~/^${version}-SNAPSHOT/)[0]
         if (qualifiedVersion == null) {
+            if (isReleaseBuild == false) {
```

```suggestion
            if (!isReleaseBuild) {
```

`isReleaseBuild` is a boolean so no need to use `== false` to force any Groovy magic. I think that at this point we can also change the definition of `isReleaseBuild` to make the type more explicit:

```java
boolean isReleaseBuild = System.getenv('RELEASE') == "1" || versionQualifier
```

---

Fix (both comments) in b9edfe49798134b418511ae9080b957e337175d5